patch_emscripten.diff

```diff
@@ -54,8 +54,8 @@
          template<typename WrapperType>
          val wrapped_extend(const std::string& name, const val& properties) {
 
---- ./emscripten/master/src/embind/embind.js	2016-12-20 13:05:20.498980029 -0800
-+++ ./emscripten/master/src/embind/embind.js	2016-12-20 13:12:11.599146929 -0800
+--- ./src/embind/embind.js	2016-12-20 13:05:20.498980029 -0800
++++ ./src/embind/embind.js	2016-12-20 13:12:11.599146929 -0800
```

Comment:
It makes the patch more generic, and usable with both emscripten/master and emscripten/incoming by changing just the patch command line (cf readme).